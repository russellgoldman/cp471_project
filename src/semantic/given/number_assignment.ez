num = num + 2 * 3;