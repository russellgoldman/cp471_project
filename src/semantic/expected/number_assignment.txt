\Tree [.program [.nextLine [.nextLinePrime [.statement
	[.expression [.iterationExpression
		[.num ]
		[.= ]
		[.num ]
		[.sumOperator [.+ ] ]
		[.multiplyExpression
			[.factor [.mutable [.2 ] ] ]
			[.multiplyExpressionPrime
				[.multiplyOperator [.* ] ]
				[.factor [.mutable [.3 ] ] ]
			]
		]
	] ]
	[.; ]
] ] ] ]